devops/templates/index.html

```diff
@@ -1,94 +1,154 @@
 {% extends "base.html" %}
 {% block content %}
-<div class="module">
-        <h1>
-            The DevOps Course: Modules
-        </h1>
 
-        <figure>
-            <img
-            src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/05/Devops-toolchain.svg/220px-Devops-toolchain.svg.png">
-            <figcaption>
-                The DevOps toolchain
-            </figcaption>
-        </figure>
+<div class="row" style="margin-top:1vh;">
+       <div class="col-sm-4">
+
+               <iframe width="350" height="315" src="https://www.youtube.com/embed/_I94-tJlovg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+                <img class="d-block w-100" src="https://cdn.pixabay.com/photo/2018/02/12/13/58/devops-3148393_1280.png" alt="Second slide">
```

Comment:
This is giving me an error image. Please keep that in mind when using images that they might not work.

Are these images you added or the professor originally wanted?